domains/thorgathis.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Thorgathis website",
+  "repo": https://github.com/Thorgathis/thorgathis.github.io",
```

Comment:
psst, you’re missing an open quotation mark (") here

Reply:
I fix